### PR TITLE
Fix links in troubleshooting section

### PR DESCRIPTION
**Pull request by IsAmrish**
Signed-off-by: isamrish <askmaurya48@gmail.com>

This PR will fix
- link in troubleshooting page
- `/troubleshooting/troubleshooting/` => `/troubleshooting/`

### CODE REVIEW

File: docs/versioned_docs/version-2.11.0/troubleshooting/troubleshooting.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
 ---
 id: troubleshooting
 title: Troubleshooting OpenEBS - Overview
+slug: /troubleshooting/
```
Comment (rakeshPRaghu) [inline docs/versioned_docs/version-2.11.0/troubleshooting/troubleshooting.md:4]: why the slug?

File: docs/versioned_docs/version-2.11.0/troubleshooting/troubleshooting.md [hunk 2]
```diff
@@ -186,4 +187,4 @@ Set the reboot timer schedule at different time i.e staggered at various interva
 
 ## See Also:
 
-[Troubleshooting Install](/docs/troubleshooting/t-install) [Troubleshooting Uninstall](/docs/troubleshooting/t-uninstall) [Troubleshooting NDM](/docs/troubleshooting/t-ndm) [Troubleshooting Jiva](/docs/troubleshooting/t-jiva) [Troubleshooting cStor](/docs/troubleshooting/t-cstor) [Troubleshooting Local PV](/docs/troubleshooting/t-localpv) [Troubleshooting Mayastor](/docs/troubleshooting/t-mayastor) [FAQs](/docs/next/faq.html) [Seek support or help](/docs/introduction/community) [Latest release notes](/docs/introduction/releases)
+[Troubleshooting Install](/docs/troubleshooting/install) [Troubleshooting Uninstall](/docs/troubleshooting/uninstall) [Troubleshooting NDM](/docs/troubleshooting/ndm) [Troubleshooting Jiva](/docs/troubleshooting/jiva) [Troubleshooting cStor](/docs/troubleshooting/cstor) [Troubleshooting Local PV](/docs/troubleshooting/localpv) [Troubleshooting Mayastor](/docs/troubleshooting/mayastor) [FAQs](/docs/next/faq.html) [Seek support or help](/docs/introduction/community) [Latest release notes](/docs/introduction/releases)
```
Comment (rakeshPRaghu) [inline docs/versioned_docs/version-2.11.0/troubleshooting/troubleshooting.md:190]:
```suggestion
[Troubleshooting Install](/docs/troubleshooting/install) [Troubleshooting Uninstall](/docs/troubleshooting/uninstall) [Troubleshooting NDM](/docs/troubleshooting/ndm) [Troubleshooting Jiva](/docs/troubleshooting/jiva) [Troubleshooting cStor](/docs/troubleshooting/cstor) [Troubleshooting Local PV](/docs/troubleshooting/localpv) [Troubleshooting Mayastor](/docs/troubleshooting/mayastor) [FAQs](/docs/additional-info/faqs) [Seek support or help](/docs/introduction/community) [Latest release notes](/docs/introduction/releases)
```
the link for faqs page, you can change into this one

File: docs/versioned_docs/version-2.11.0/troubleshooting/troubleshooting.md (hunk 1 above)
Comment (IsAmrish) [reply docs/versioned_docs/version-2.11.0/troubleshooting/troubleshooting.md:4]: for converting this `https://gallant-pasteur-5e438c.netlify.app/docs/troubleshooting/troubleshooting` to `https://gallant-pasteur-5e438c.netlify.app/docs/troubleshooting/`

**Review by AVRahul — APPROVED** [review]

File: docs/versioned_docs/version-2.11.0/troubleshooting/troubleshooting.md (hunk 1 above)
Comment (rakeshPRaghu) [reply docs/versioned_docs/version-2.11.0/troubleshooting/troubleshooting.md:4]: oh ok! understood

**Review by rakeshPRaghu — APPROVED** [review]

**Pull request merged**